test/analysis/typeComplexityAnalysis.test.ts

```diff
@@ -797,20 +799,6 @@ describe('Test getQueryTypeComplexity function', () => {
             });
         });
 
-        /**
-         * FIXME: handle lists of unknown size. change the expected result Once we figure out the implementation.
-         */
-        xtest('with lists of unknown size', () => {
-            query = `
```

Comment:
Could this not be used? Expect the result to be `11` (1 query + 10 searchresults)